fenix/app/src/main/java/org/mozilla/fenix/home/HomeMenuView.kt

```diff
@@ -76,6 +76,14 @@ class HomeMenuView(
                 ThemeManager.resolveAttribute(R.attr.textPrimary, context),
             ),
         )
+
+        menuButton.get()?.register(
+            object : mozilla.components.concept.menu.MenuButton.Observer {
+                override fun onShow() {
+                    Events.toolbarMenuVisible.record(NoExtras())
```

Comment:
❓ I see that `emitOpenMenuFact` also record the same event.  Will this cause a problem?

Reply:
That's a good question!

`emitOpenMenuFact` doesn't work from the home page, because the function exists in the [wrapper](https://searchfox.org/mozilla-mobile/source/firefox-android/android-components/components/browser/toolbar/src/main/java/mozilla/components/browser/toolbar/display/MenuButton.kt#25) class for menu item, but the home page skips the wrapper and uses the inner class directly, which the [class](https://searchfox.org/mozilla-mobile/source/firefox-android/android-components/components/browser/menu/src/main/java/mozilla/components/browser/menu/view/MenuButton.kt#35) itself says not to do. 

So as of now we are not collecting any menu data from the home page, hence I am adding it additionaly. Now that I think about it, if it wasn't an intentional side effect (disabled metrics) maybe we should refactor the way menu is used on the home page.